packages/ui-progress/src/ProgressCircle/__examples__/ProgressCircle.examples.tsx

```diff
@@ -31,28 +32,23 @@ export default {
     valueNow: [0, 40, 80, 100],
     renderValue: [
       null,
-      // @ts-expect-error ts-migrate(7031) FIXME: Binding element 'valueNow' implicitly has an 'any'... Remove this comment to see the full error message
-      // eslint-disable-next-line react/display-name
-      ({ valueNow, valueMax }) => (
+      ({ valueNow, valueMax }: { valueNow: number; valueMax: number }) => (
         <span>{Math.round((valueNow / valueMax) * 100)}</span>
       )
     ]
   },
-  // @ts-expect-error ts-migrate(6133) FIXME: 'props' is declared but its value is never read.
-  getComponentProps: (props) => {
+  getComponentProps: () => {
     return {
       screenReaderLabel: 'Passing students',
       valueMax
     }
   },
-  // @ts-expect-error ts-migrate(7006) FIXME: Parameter 'props' implicitly has an 'any' type.
-  getExampleProps: (props) => {
+  getExampleProps: (props: ProgressCircleProps) => {
     return {
-      background: props.color.includes('inverse')
+      background: props?.color?.includes('inverse')
```

Comment:
Just for consistency's stake: In ProgressBar you typed it as `props.color!.includes()`, because color is always set by defaultProps, so can be like this here too :)